app/serializers/api/v1/update_serializer.rb

```diff
@@ -0,0 +1,10 @@
+module Api
+  module V1
+    class UpdateSerializer < ActiveModel::Serializer
+      attribute :category
+      attribute :link
+      attribute :description
+      attribute :updated_at, key: :date
```

Comment:
should we format the date here? or is it OK for the front to do it? (We don't need the time)

Reply:
true, no need for FE to do it